client/webserver/site/src/js/forms.js

```diff
@@ -9,22 +9,56 @@ export function bindNewWallet (app, form, success) {
   // CREATE DCR WALLET
   // This form is only shown the first time the user visits the /register page.
   const fields = Doc.parsePage(form, [
-    'iniPath', 'acctName', 'newWalletPass', 'submitCreate', 'walletErr',
+    'acctName', 'newWalletPass',
+    'cfgFile', 'selectedCfgFile', 'removeCfgFile', 'addCfgFile',
+    'submitCreate', 'walletErr',
     'newWalletLogo', 'newWalletName', 'wClientPass'
   ])
+
+  // config file upload
+  const defaultCfgFileText = 'none selected'
+  fields.selectedCfgFile.textContent = defaultCfgFileText
+  const resetCfgFileUpload = () => {
+    fields.cfgFile.value = ''
+    fields.selectedCfgFile.textContent = defaultCfgFileText
+    Doc.hide(fields.removeCfgFile)
+    Doc.show(fields.addCfgFile)
```

Comment:
We have the `asset.info.configpath`. I wonder if we can check for a file there and set it for the user if the file exists.

Reply:
Given the coming changes to the add wallet form, this shouldn't be necessary. If by default, we show the generated wallet config fields in the form, we can expect that if the user chooses to upload a file rather than use the form fields, he'd set a file.